eng/scripts/packagevalidation/PrePublish-Validation.ps1

```diff
@@ -55,6 +55,16 @@ if (($ArtifactName -ne $PackageId) -or ($PackageName -ne "$PackageId.$PackageVer
 }
 
 LogDebug "Validating Package Configuration..."
+pushd $PSScriptRoot
+dotnet tool install dotnet-script
+foreach ($file in $PackageDlls) {
```

Comment:
You should add a comment as to why we are using another process instead of doing nit in pwsh itself.

Reply:
I added that below.